### PR TITLE
Remove all references to help@actualbudget.com

### LINKED ISSUE #625: [Enhancement]: Remove all references to help@actualbudget.com
### Verified issue does not already exist?

- [X] I have searched and found no existing issue

### What happened?

Remove all references to help@actualbudget.com and replace with https://github.com/actualbudget/actual/issues

### PR DESCRIPTION
**Pull request by rich-howell**
Replaced help@actualbudget.com with link to GitHub issues, this should resolve #625

### CODE REVIEW

**Comment by j-f1** [comment]
Can you link to https://actualbudget.github.io/docs/Contact instead? I don’t want folks to get stuck if they’ve never used GitHub before.

**Comment by j-f1** [comment]
Confirmed that there don’t appear to be any other email addresses in the app 👍

File: packages/desktop-client/src/components/accounts/AccountSyncCheck.js [hunk 1]
```diff
@@ -42,7 +42,13 @@ function getErrorMessage(type, code) {
     default:
   }
 
-  return 'An internal error occurred. Try to login again, or contact help@actualbudget.com for support.';
+  return (
+    <>
+      'An internal error occurred. Try to login again, or get
+      <a href="https://actualbudget.github.io/docs/Contact/">intouch</a>
+      for support.''
```
Comment (j-f1) [inline packages/desktop-client/src/components/accounts/AccountSyncCheck.js:49]:
```suggestion
      An internal error occurred. Try to login again, or get{' '}
      <a href="https://actualbudget.github.io/docs/Contact/">in touch</a>{' '}
      for support.
```

**Comment by j-f1** [comment]
ohhhh… it looks like `loot-core` doesn’t have the JSX transform enabled. Maybe the `errors` returned by `accounts-sync` should have a non-human-readable property named `kind` or `value` or something, along with a `name` property for the account name? Then the UI could handle switching through the valid types of error returned here and generate the human-readable message including link tags?

**Comment by rich-howell** [comment]
> ohhhh… it looks like `loot-core` doesn’t have the JSX transform enabled. Maybe the `errors` returned by `accounts-sync` should have a non-human-readable property named `kind` or `value` or something, along with a `name` property for the account name? Then the UI could handle switching through the valid types of error returned here and generate the human-readable message including link tags?

I have removed it for now and just put a plain text message in there.

File: packages/desktop-client/src/components/accounts/AccountSyncCheck.js [hunk 2]
```diff
@@ -42,7 +42,13 @@ function getErrorMessage(type, code) {
     default:
   }
 
-  return 'An internal error occurred. Try to login again, or contact help@actualbudget.com for support.';
+  return (
+    <>
+      An internal error occurred. Try to login again, or get{' '}
+      <a href="https://actualbudget.github.io/docs/Contact/">in touch</a> for
```
Comment (MatissJanis) [inline packages/desktop-client/src/components/accounts/AccountSyncCheck.js:48]: **💬 suggestion:** ‏open in a new tab
```suggestion
      <a href="https://actualbudget.github.io/docs/Contact/" target="_blank">in touch</a> for
```

File: packages/desktop-client/src/components/tutorial/Final.js [hunk 3]
```diff
@@ -20,11 +20,8 @@ function Final({ targetRect, navigationProps }) {
 
       <P>
         You probably want to delete the transactions you added and clean up your
-        budget. If you have any questions or feedback, email{' '}
-        <ExternalLink asAnchor href="mailto:help@actualbudget.com">
-          help@actualbudget.com
-        </ExternalLink>
-        .
+        budget. If you have any questions or feedback, please get{' '}
+        <a href="https://actualbudget.github.io/docs/Contact/">in touch</a>.
```
Comment (MatissJanis) [inline packages/desktop-client/src/components/tutorial/Final.js:24]:
```suggestion
        <a href="https://actualbudget.github.io/docs/Contact/" target="_blank">in touch</a>.
```

**Review by MatissJanis — APPROVED** [review]
Two nitpicks. Let me know what you think.

Apart from that: LGTM 👍

**Review by MatissJanis — APPROVED** [review]

**Pull request merged**